doc/source/ray-core/tasks/scheduling.rst

```diff
@@ -22,11 +22,28 @@ See :ref:`Placement Group <ray-placement-group-doc-ref>` for more details.
 Scheduling Strategy
 -------------------
 Tasks support ``scheduling_strategy`` option to specify the strategy used to decide the best node among available nodes.
-Currently the supported strategies are "DEFAULT" and "SPREAD".
+Currently the supported strategies are ``"DEFAULT"``, ``"SPREAD"`` and
+``ray.util.scheduling_strategies.NodeSchedulingStrategy(node_id, soft)``.
+
 "DEFAULT" is the default strategy used by Ray. With the current implementation, Ray will try to pack tasks on nodes
 until the resource utilization is beyond a certain threshold and spread tasks afterwards.
+
 "SPREAD" strategy will try to spread the tasks among available nodes.
 
+NodeSchedulingStrategy is the low level strategy that allows a task to be scheduled onto a particular node specified by the hex node id.
+The ``soft`` flag specifies whether the task is allowed to run somewhere else if the specified node doesn't exist (e.g. the node dies)
+or is infeasible for the required resources. If it is True, the task will be scheduled onto other feasible nodes under such case.
```

Comment:
```suggestion
or is infeasible because it does not have the resources required to run the task. In these cases, if ``soft`` is True, the task will be scheduled onto a different feasible node.
```